Program.cs

```diff
@@ -49,10 +49,11 @@ static void Main(string[] args)
 
             try
             {
-                foreach (var arg in args)
+                for (int i = 0; i < args.Length; i++)
                 {
-                    parameter = arg.Substring(0, arg.IndexOf('='));
-                    parameterValue = arg.Substring(arg.LastIndexOf('=') + 1);
+                    parameter = args[i];
```

Comment:
args[i] is safe. Args i+1 would go out of range for the last element?

Reply:
I am using for loop instead of for each. if needed i will add a check. As per the given requirement we need to remove the '=' sign from the arguments. so we have a space between argument name and value. 
Sample IP: "-mode" "modeValue" "-cloudurl" "https://cloudurl/" "-appid" "appidValue" "-APIkey"  "ApiKeyValue" "-uploadpath" "uploadpathValue" "-handledDirectory" "handledDirectoryValue"